src/providers/LanguageProvider.tsx

```diff
@@ -23,6 +23,9 @@ i18n.loadLocaleData({
   fr: { plurals: fr },
 })
 
+i18n.load(DEFAULT_LOCALE, defaultLocale.messages)
```

Comment:
This dude said adding these lines fixed his problem: https://github.com/lingui/js-lingui/issues/1194#issuecomment-1068488619